### DISCUSSION TITLE
v3.2.106

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#14010] [General]
## What's Changed
* **v3.2.104 & v3.2.105:** Attempts to resolve issues with lazy widgets introduced by changes to the section component in v3.2.103.
* Fixes issue with lazy relation managers introduced by v3.2.105, by reverting the previous fixes alongside the change made to the section component in v3.2.103.
* fix: Clearing bulk actions leaves checkboxes by @danharrin in https://github.com/filamentphp/filament/pull/14004

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.2.105...v3.2.106

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.2.106'>v3.2.106</a>.</em>

**Comment by NightingaleWK** [comment]
This morning found that the relationship manager could not create, delete and other operations, and then fixed, thank you
